src/test/java/it/pn/frontend/e2e/stepDefinitions/destinatario/personaFisica/LoginPersonaFisicaPagoPA.java

```diff
@@ -49,11 +48,12 @@ public void loginPageDestinatarioVieneVisualizzata(String datipersonaFisica) {
                     Assert.fail("Non stato possibile trovare l'ambiente inserito, Inserisci in -Denvironment test o dev o uat");
         }
     }
-        @Given("Login Page persona fisica test viene visualizzata")
-        public void loginPageDestinatarioVieneVisualizzataConUrl() {
 
-           String url = "https://cittadini.test.notifichedigitali.it/";
-            this.driver.get(url);
+    @Given("Login Page persona fisica test viene visualizzata")
+    public void loginPageDestinatarioVieneVisualizzataConUrl() {
+
+        String url = "https://cittadini.test.notifichedigitali.it/";
```

Comment:
view https://github.com/pagopa/pn-frontend-e2e/pull/176 and proceed similarly

Reply:
i didn't change anything about this part, just formatted